### PR TITLE
Qt/GraphicsSettings: Save Adapter index

### PR DESCRIPTION
**Pull request by spycrab**
Fixes [issue #11043](https://bugs.dolphin-emu.org/issues/11043).

### CODE REVIEW

File: Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp [hunk 1]
```diff
@@ -172,6 +172,10 @@ void GeneralWidget::LoadSettings()
 
 void GeneralWidget::SaveSettings()
 {
+#ifdef _WIN32
```
[the remark this answers is not in the document]
Comment (spycrab) [reply Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: "Multiadapter Support ist only present on Windows"?

[the remark this answers is not in the document]
Comment (lioncash) [reply Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: Yeah, that works

[the remark this answers is not in the document]
Comment (spycrab) [reply Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: Done.

File: Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp [hunk 2]
```diff
@@ -64,6 +64,7 @@ void GeneralWidget::CreateWidgets()
   m_video_layout->addWidget(new QLabel(tr("Backend:")), 0, 0);
   m_video_layout->addWidget(m_backend_combo, 0, 1);
 
+  // Multiadapter support is only present on Windows
```
Comment (stenzek) [inline Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp]: Not actually the case, Vulkan on Linux supports multiple adapters.

**Comment by hammer300** [comment]
Cant set vulkan graphics adater on latest 7157.  crashes emulator.  long time user i figured you guys all ready knew just wanted to give a heads up.

**Comment by hammer300** [comment]
nevermind i was wrong apparently, there is a nvidia driver bug on the latest 4-25-18 release.   397.31

**Pull request merged**